src/plugins/friendsSince/index.tsx

```diff
@@ -4,50 +4,40 @@
  * SPDX-License-Identifier: GPL-3.0-or-later
  */
 
-import { classNameFactory } from "@api/Styles";
 import ErrorBoundary from "@components/ErrorBoundary";
 import { Devs } from "@utils/constants";
 import { getCurrentChannel } from "@utils/discord";
 import { Logger } from "@utils/Logger";
-import { classes } from "@utils/misc";
 import definePlugin from "@utils/types";
-import { findByCodeLazy, findByPropsLazy } from "@webpack";
-import { Heading, React, RelationshipStore, Text } from "@webpack/common";
+import { findByCodeLazy, findByPropsLazy, findLazy } from "@webpack";
+import { Heading, RelationshipStore, Text } from "@webpack/common";
+
+const containerWrapper = findByPropsLazy("memberSinceWrapper");
+const container = findByPropsLazy("memberSince");
 
-const container = findByPropsLazy("memberSinceWrapper");
 const getCreatedAtDate = findByCodeLazy('month:"short",day:"numeric"');
 const locale = findByPropsLazy("getLocale");
-const lastSection = findByPropsLazy("lastSection");
-
-const cl = classNameFactory("vc-friendssince-");
+const section = findLazy((m: any) => Object.values(m).length === 1 && m.section !== void 0);
```

Comment:
flipping the condition will be much more performant since it avoids thousands of unnecessary Object.values() calls
```suggestion
const section = findLazy((m: any) => m.section && Object.values(m).length === 1);
```